docs/eventsources/ha.md

```diff
@@ -53,6 +53,16 @@ old one is gone.
 - Redis
 - Resource
 
+## Kubernetes Leader Election
+
+By default, Argo Events will use NATS for the HA leader election. Alternatively,
+you can opt-in to a Kubernetes native leader election (that uses a Lease) by
+specifying the following annotation.
+```yaml
+annotations:
+  events.argoproj.io/leader-election: k8s
+```
```

Comment:
Thanks! Also need to mention the service account used for the EventSource need to have extra RBAC config, and give the example `Role` and `RoleBinding`.

Reply:
Yes! Thank you, added :+1: